src/scene/shader-lib/chunks/lit/frag/TBNfast.js

```diff
@@ -1,5 +1,5 @@
 export default /* glsl */`
-void getTBN() {
-    dTBN = mat3(dTangentW, dBinormalW, dVertexNormalW);
+void getTBN(vec3 tangent, vec3 binormal, vec3 normal) {
+    dTBN = mat3(tangent, binormal, normal);
```

Comment:
Is this chunk actually used anywhere? Still worthwhile keeping?

Reply:
It's switched on with a material parameter. We could definitely delete this one, but would prefer to do so in a separate PR since it changes the material API.